scripts/diffmazing_cropper.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+#
+# Converts diff images from Cypress into snapshots we can commit to Git.
+
```

Comment:
Always a good idea to set errexit, nounset and pipefail in bash scripts (long versions preferred for readability): 
```suggestion
set -o errexit
set -o nounset
set -o pipefail
```

Couple of useful references:
* https://news.ycombinator.com/item?id=10736584
* https://medium.com/factualopinions/consider-starting-all-your-bash-scripts-with-these-options-74fbec0cbb83